src/Artsy/Router/RouterLink.tsx

```diff
@@ -1,23 +1,44 @@
 import { Link, LinkProps } from "found"
+import PropTypes from "prop-types"
 import React from "react"
 
 /**
- * Wrapper component around found's <Link> component that can be easily styled
- * via styled-components; e.g.,
+ * Wrapper component around found's <Link> component with a fallback to a normal
+ * <a> tag if ouside of a routing context.
+ *
+ * NOTE: that if using styled-components, <RouterLink> can be easily styled
+ * like so:
  *
  * const StyledLink = styled(RouterLink)`
  *   ...
  * `
  */
-export const RouterLink: React.FC<LinkProps> = ({
-  to,
-  className,
-  children,
-  ...props
-}) => {
-  return (
-    <Link to={to} className={className} {...props}>
-      {children}
-    </Link>
-  )
+
+export const RouterLink: React.FC<LinkProps> = (
+  { to, className, children, ...props },
+  context
+) => {
+  const isRouterContext = Boolean(context.router)
+
+  if (isRouterContext) {
+    return (
+      <Link to={to} className={className} {...props}>
+        {children}
+      </Link>
+    )
+  } else {
+    return (
+      <a href={to as string} className={className} {...props}>
```

Comment:
If `className` is just a pass-through in both branches here, does it need to be destructured out?